english/explore-outside/journey-to-flaming-tower/jellybus/too-socially-awkward/too-socially-awkward.md

```diff
@@ -0,0 +1,9 @@
+You decide to keep walking.  You don't do well around strangers.  You're very shy.  Soon after, 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.